### PR TITLE
Add netlify configuration files

### PR DESCRIPTION
**Pull request by mewtaylor**
In order to consolidate information for deployment in one place (to make it easier to replicate this), we're adding these 3 files to the repo. Environment variables that are sensitive are still specific in the netlify configuration itself

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[build]
+  publish = "public"
+  command = "yarn build"
+[build.environment]
+  YARN_FLAGS = "--no-ignore-optional"
```
Comment (esetnik) [inline netlify.toml:5]: Can you add YARN_VERSION = “1.22.4”

**Comment by mmoutenot** [comment]
Note that you will still need

`GATSBY_API_KEY`
and
`GATSBY_SPREADSHEET_ID` set in the environment variables section of Netlify's Build settings.

https://app.netlify.com/sites/{SITE_ID}/settings/deploys#environment-variables

File: netlify.toml (hunk 1 above)
Comment (mewtaylor) [reply netlify.toml:6]: @esetnik added!

**Comment by esetnik** [comment]
@mewtaylor i don’t see netlify build status on this PR. Can you check it’s already linked to the project here?

File: netlify.toml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[build]
+  publish = "public"
+  command = "yarn build"
+[build.environment]
+  YARN_VERSION = “1.22.4”
```
Comment (esetnik) [inline netlify.toml:5]: Wrong quote type here. Need ".

**Pull request merged**